purchase_order_line_description/tests/test_purchase_order_line_description.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2015 Alex Comba - Agile Business Group
+# Copyright 2017 Tecnativa - vicent.cubells@tecnativa.com
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields
+from odoo.tests import common
+
+
+class TestPurchaseOrderLineDescription(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestPurchaseOrderLineDescription, cls).setUpClass()
+        partner = cls.env["res.partner"].create({"name": "Test partner"})
+        cls.product = cls.env["product.product"].create(
+            {
+                "name": "Product",
+                "standard_price": 10,
+                "description_purchase": "description for purchase",
+            }
+        )
+        group_id = cls.env.ref(
```

Comment:
As in coding guideline
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#342variable-names
```suggestion
        group = cls.env.ref(
```